.github/workflows/ci.yaml

```diff
@@ -421,6 +421,43 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
 
+  deploy-demo-env:
+    name: Deploy updated images to the demo environment
+    runs-on: self-hosted
+    if: github.event_name == 'release' || (github.event_name == 'push' && github.ref_name == 'main') || github.event_name == 'workflow_dispatch'
+    env:
+      IMAGE_REPOSITORY: ghcr.io/${{ github.repository_owner }}/trento-web
+      IMAGE_TAG: "${{ (github.event_name == 'release' && github.event.release.tag_name) || (github.event_name == 'push' && github.ref_name == 'main' && 'rolling') || github.sha }}"
+    needs: build-and-push-container-images
+    steps:
+      - name: Start a local k8s cluster
+        uses: jupyterhub/action-k3s-helm@v3
+        with:
+          k3s-channel: latest
+
+      - name: Add bitnami helm deps
+        run: |
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm repo update
+
+      - name: Download and unzip helm chart
+        run: |
+          rm rolling.zip | true
```

Comment:
I'm having some doubts about this: should each deployment fetch the rolling helm charts or should we make exceptions for the releases so that they also attempt to install the helm charts that correspond to that same release?

IMO going always for the `rolling` tag is safer but I'd like to have more opinions on it

Reply:
By don't use simply use `$IMAGE_TAG`. It would actually do the same, but it looks more elegant at least. I guess the archives with the tag names are created. Otherwise, you can always use `main.zip` instead of rolling as well,

Reply:
The trouble then would be that it would enforce a requirement that the helm chart version x.y.z is already released when version x.y.z gets released here. The rolling is a more generic one